demos/gfql/ten_mins_to_gfql.ipynb

```diff
@@ -0,0 +1,2376 @@
+       "\n",
+       "                      filer_org_name begin_date   end_date  \\\n",
+       "0  The Bank of New York Mellon Corp. 2015-03-25 2015-09-25   \n",
+       "1  The Bank of New York Mellon Corp. 2015-03-30 2015-09-25   \n",
+       "2  The Bank of New York Mellon Corp. 2012-07-05 2012-07-05   \n",
+       "3  The Bank of New York Mellon Corp. 2012-06-20 2012-06-20   \n",
+       "4  The Bank of New York Mellon Corp. 2012-05-31 2012-05-31   \n",
+       "\n",
+       "        originator_bank_id          originator_bank originator_bank_country  \\\n",
+       "0         cimb-bank-berhad         CIMB Bank Berhad               Singapore   \n",
+       "1         cimb-bank-berhad         CIMB Bank Berhad               Singapore   \n",
+       "2  barclays-bank-plc-ho-uk  Barclays Bank Plc Ho UK          United Kingdom   \n",
+       "3  barclays-bank-plc-ho-uk  Barclays Bank Plc Ho UK          United Kingdom   \n",
+       "4  barclays-bank-plc-ho-uk  Barclays Bank Plc Ho UK          United Kingdom   \n",
+       "\n",
+       "  originator_iso                                beneficiary_bank_id  \\\n",
+       "0            SGP               barclays-bank-plc-london-england-gbr   \n",
+       "1            SGP               barclays-bank-plc-london-england-gbr   \n",
+       "2            GBR  skandinaviska-enskilda-banken-stockholm-sweden...   \n",
+       "3            GBR  skandinaviska-enskilda-banken-stockholm-sweden...   \n",
+       "4            GBR  skandinaviska-enskilda-banken-stockholm-sweden...   \n",
+       "\n",
+       "                beneficiary_bank beneficiary_bank_country beneficiary_iso  \\\n",
+       "0              Barclays Bank Plc           United Kingdom             GBR   \n",
+       "1              Barclays Bank Plc           United Kingdom             GBR   \n",
+       "2  Skandinaviska Enskilda Banken                   Sweden             SWE   \n",
+       "3  Skandinaviska Enskilda Banken                   Sweden             SWE   \n",
+       "4  Skandinaviska Enskilda Banken                   Sweden             SWE   \n",
+       "\n",
+       "   number_transactions  amount_transactions  \n",
+       "0                 68.0         5.689852e+07  \n",
+       "1                118.0         1.162384e+08  \n",
+       "2                  NaN         5.000000e+03  \n",
+       "3                  NaN         9.990000e+03  \n",
+       "4                  NaN         1.200000e+04  "
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "g._edges.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualization Setup\n",
+    "\n",
+    "Create a color palette for visualizing the graph."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def nonlinear_palette_generator(v, linear_palette):\n",
```

Comment:
recommend not doing custom palettes in demos, and/or moving this to a feature, as this has negative pedagogic value... like stay as minimal as you can to teach the concepts